migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -351,4 +351,18 @@ INSERT INTO permit_amendment_status_code
 VALUES
     ('ACT', 'Active', 10, 'system-mds', 'system-mds'),
     ('RMT', 'Remitted', 20, 'system-mds', 'system-mds')
-ON CONFLICT DO NOTHING;
\ No newline at end of file
+ON CONFLICT DO NOTHING;
+
+
+INSERT INTO mds_required_document (
+    req_document_name,
+    req_document_description,
+    req_document_category_guid,
+    req_document_due_date_type,
+    req_document_due_date_period_months,
+    create_user,
+    update_user
+    )
+VALUES
+    ('As Built Reports', '10.4.5(c)', '6ab98b9a-0e66-4f26-99de-e3c270dea7b6', 'FIS', 12, 'system-mds','system-mds')
```

Comment:
Where is this guid created? Is there any way around hard-coding it? This is a randomly-generated value, and I don't have a great explanation for why this is a problem, but it feels bad.

Thoughts / explanation on why it's done this way?

Reply:
It was done this way because we wanted a look up-table for category. This has been added to backlog and brought to Aaron's attention. Totally agree, and that structural change is coming up (hopefully soon). 

https://bcmines.atlassian.net/browse/MDS-1533

Reply:
Okay. Glad it's captured. Cost of doing it later vs just making that change in this PR?

This PR seems to be adding another place that will need to be updated in the future. If it isn't a huge change to make now, it might be appropriate to do that now?

Reply:
it's a significant changes relative to the original request from Aaron (add a document to the list). 

This is just a row in the table, and doesn't add to the complexity of the existing pattern.